generator/config/stage/group.yaml

```diff
@@ -18,5 +18,105 @@ arguments:
         type:
             - accumulator
         variadic: object
+        variadicMin: 0
         description: |
             Computed using the accumulator operators.
+tests:
+    -
+        name: 'Count the Number of Documents in a Collection'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/group/#count-the-number-of-documents-in-a-collection'
+        pipeline:
+            -
+                $group:
+                    _id: ~
+                    count:
+                        $count: {}
+    -
+        name: 'Retrieve Distinct Values'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/group/#retrieve-distinct-values'
+        pipeline:
+            -
+                $group:
+                    _id: '$item'
```

Comment:
This example shows that an accumulator is not required. I reduced `variadicMin` to 0.